src/app/view/components/_common/gravatar/gravatar.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input } from '@angular/core';
+import { GravatarService } from '../../../../api/services/gravatar-service';
+let md5 = require('md5');
+
+@Component({
+    selector: 'dot-gravatar',
+    styles: [require('./gravatar.component.scss')],
+    templateUrl: './gravatar.component.html'
+})
+
+export class GravatarComponent {
+    @Input() email;
+    @Input() size;
+    public gravatarProfileStyles;
+    public gravatarProfile;
+    public dotAvatarProfile;
+    public dotAvatarProfileStyles;
+
+    constructor(private gravatarService: GravatarService) {
+
+    }
+
+    ngOnInit(): void {
+        const hash = md5(this.email);
+        const profile$ = this.gravatarService.loadGravatarProfile(hash);
+        profile$.subscribe(
+            data => {
+                this.gravatarProfileStyles = {
+                    'background': 'url("' + data.entry[0].photos[0].value + '?s=' + this.size + '")',
+                    'border-radius': this.size / 2 + 'px',
```

Comment:
You can do the ```border-radius: 50%``` in the scss file and don't have to do the math and put it inline here.